guide/onboarding/funding-a-wallet.md

```diff
@@ -22,101 +30,300 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+## Do your users own bitcoin?
 
-{% include fact/pros.html %}
- - Privacy
- - No exchange fees
- - Simplifies onboarding
- {% include fact/close.html %}
+The first step in having users fund your wallet is finding out if they currently own bitcoin or not. This will determine what kind of information to present to your users next as well as guide them towards how they will fund their wallet. If the user wants to fund their wallet at a later time they should have the option to skip this process and jump straight into the wallet.
 
-{% include fact/cons.html %}
- - Managing taxes
- - Difficult to find businesses/individuals willing to pay in bitcoin
- - Limited options for recurring payments
- {% include fact/close.html %}
+If your user does not have any bitcoin, your application should guide them in obtaining some. This could be done by recommending users find a local exchange to buy bitcoin from and to come back once they have some before proceeding. Alternatively, you could offer the option to buy directly within your application to speed up this process.
 
-### Buying bitcoin
+<div class="image-slide-gallery">
 
-<div class="center" markdown="1">
+{% include picture.html
+   image = "/assets/images/guide/onboarding/funding-a-wallet/own-bitcoin.png"
+   retina = "/assets/images/guide/onboarding/funding-a-wallet/own-bitcoin@2x.png"
+   layout = "shadow"
+   caption = "Ask the user if they own any bitcoin."
+   alt-text = "Screen asking the user if they own any bitcoin."
+   width = 250
+   height = 541
+%}
 
-{% include image.html
+{% include picture.html
+   image = "/assets/images/guide/onboarding/funding-a-wallet/where-to-buy.png"
+   retina = "/assets/images/guide/onboarding/funding-a-wallet/where-to-buy@2x.png"
+   layout = "shadow"
+   caption = "If not, guide them on where to get some."
+   alt-text = "Screen guiding user where to buy bitcoin."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
    image = "/assets/images/guide/onboarding/funding-a-wallet/buy-bitcoin.png"
    retina = "/assets/images/guide/onboarding/funding-a-wallet/buy-bitcoin@2x.png"
+   layout = "shadow"
+   caption = "Alternatively, offer the option to buy bitcoin directly in your application."
+   alt-text = "Screen showing option to buy within the application"
+   width = 250
+   height = 541
+%}
+
+</div>
+
+
+## How it works
+
+If the user has purchased some bitcoin either outside or within your application, or they already own some, the next thing should be explaining how their wallet works once its funded. 
+
+Once users funds their wallet and they have a channel open they will initially only be able to send Lightning payments. This is due to the payment channel the user opens in later steps only having outbound liquidity. This can also be an opportunity to give a soft introduction to what a payment channel is and how they work. This should be clear to users before they fund the wallet to avoid future friction.
```

Comment:
@GBKS The whole "How it works" section confused me a lot on my first read-through. I felt the same way @GBKS; I wasn't clear if the channel opening had already occurred or was going to occur in the future.

Having read the page multiple times, I think @Bosch-0 is trying to convey a scenario similar to Blixt where you fund your wallet on-chain and then a channel is automatically opened once your funds are cleared on-chain. Thus, it's most proper to explain to the user what is going to happen to their money.

So I see the logic in needing to explain to the reader what is going to happen in a later step. Perhaps it could be more concise or worded differently.

Reply:
Yes @sbddesign is correct with what I was aiming to do here, I will make this section more concise as to avoid confusion.